systers_portal/common/tests/selenium/base.py

```diff
@@ -0,0 +1,87 @@
+from selenium import webdriver
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from meetup.models import (Meetup, MeetupLocation, RequestMeetupLocation)
+from django.contrib.auth.models import User
+from django.utils import timezone
+from cities_light.models import City, Country
+from users.models import SystersUser
+from django.conf import settings
+from django.contrib.auth import (SESSION_KEY, BACKEND_SESSION_KEY,
+                                 HASH_SESSION_KEY)
+from django.contrib.sessions.backends.db import SessionStore
+import json
+import os
+
+browsers = {
+    'firefox': webdriver.Firefox,
+    'chrome': webdriver.Chrome,
+}
+
+
+class SeleniumTestCase(StaticLiveServerTestCase):
+    """
+    A base test case for selenium, providing different helper methods.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.browser = browsers['firefox']()
+        cls.browser.implicitly_wait(0)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.browser.quit()
+        super().tearDownClass()
+
+    def setUp(self):
+        country = Country.objects.create(name='Bar', continent='AS')
+        self.location = City.objects.create(
+            name='Foo', display_name='Foo', country=country)
+        with open('%s%s' %
```

Comment:
Please write a comment here mentioning why did you import the credentials.

Reply:
in fact, these kind of '%s%s' usages are error prone its better to use proper string formatting like - 
`'{0}{1}'.format(arg1, arg2)`, even though in latest py release - they use fstrings (which are the best).

Reply:
@tonythomas01 @ajatprabha alright.